### PR TITLE
Feature/aspect changes

### PR DESCRIPTION
**Pull request by mariusz89016**
Change ID type (int -> UUID) & clean up code (a little bit)

### CODE REVIEW

File: core/src/main/scala/akka/MethodBang_template.scala [hunk 1]
```diff
@@ -20,63 +19,42 @@ class MethodBang {
...
   def aspectAroundReceive(joinPoint: ProceedingJoinPoint): AnyRef = {
     val actor = joinPoint.getArgs()(0)
-//    println(actor.getClass)
     val receive = joinPoint.getArgs()(1)
     val message = joinPoint.getArgs()(2) match {
       case msgWrapper: MessageWrapper =>
-        println("RECEIVED: " + msgWrapper.id + " -> " + msgWrapper.msg)
         actor match {
-          case act: DistributedStackTrace => act.ZMIENNA = msgWrapper.id
+          case act: TracedActor => act.MessageWrapperId = msgWrapper.id
```
Comment (salceson) [inline core/src/main/scala/akka/MethodBang_template.scala]: `case tracedActor: TracedActor` - maybe better?

File: core/src/main/scala/pl/edu/agh/iet/akka_debugging/DistributedStackTraceUtil.scala [hunk 2]
```diff
@@ -7,10 +9,10 @@ trait DistributedStackTraceMessage {
   val stackTrace = Thread.currentThread().getStackTrace
 }
 
-trait DistributedStackTrace {
+trait TracedActor {
   self: Actor =>
 
-  var ZMIENNA = -1
+  var MessageWrapperId: UUID = UUID.fromString("00000000-0000-0000-0000-000000000000")
```
Comment (salceson) [inline core/src/main/scala/pl/edu/agh/iet/akka_debugging/DistributedStackTraceUtil.scala:15]: Shouldn't it be annotated with `@volatile` - just asking? 

BTW: finally some proper name :+1:

**Comment by salceson** [comment]
Just some little changes but otherwise good work, @mariusz89016!

File: core/src/main/scala/pl/edu/agh/iet/akka_debugging/DistributedStackTraceUtil.scala (hunk 2 above)
Comment (mariusz89016) [inline core/src/main/scala/pl/edu/agh/iet/akka_debugging/DistributedStackTraceUtil.scala:15]: I don't know - but I think rather no (I think that this advice is run on the same thread as instrumented method)

**Comment by salceson** [comment]
Ok, I think that it's ok to merge now.

**Pull request merged**